Find a functionally equivalent code:android.graphics.Canvas.getMatrix:COMMENT
Method Modifier: public      deprecated  
Comment:/**
 * Return, in ctm, the current transformation matrix. This does not alter
 * the matrix in the canvas, but just returns a copy of it.
 */

@@ -1,3 +1,3 @@
 {
-    native_getCTM(mNativeCanvas, ctm.native_instance);
+    native_getCTM(mNativeCanvasWrapper, ctm.native_instance);
 }

